modules/core/src/lib/deck.ts

```diff
@@ -181,6 +181,8 @@ export type DeckProps = {
   _pickable?: boolean;
   /** (Experimental) Fine-tune attribute memory usage. See documentation for details. */
   _typedArrayManagerProps?: TypedArrayManagerOptions;
+  /** An array of Widget instances to be added to the parent element. */
+  widgets?: Widget[];
```

Comment:
The PR description states that this would be an experimental prop?
```suggestion
  _widgets?: Widget[];
```

Reply:
Oh, I forgot to change that. After discussion we're not going to use experimental. The plan is for stable support in v9